torch/quantization/fx/_equalize.py

```diff
@@ -61,9 +61,8 @@ def __init__(self, dtype=torch.quint8, qscheme=torch.per_tensor_affine,
         self.equalization_scale = torch.empty(0)
 
     def forward(self, x_orig):
-        # TODO: Allow for convoluational layers
-        if not (x_orig.ndim == 2):
-            raise ValueError("InputEqualizationObserver only supports Linear layers")
+        if not (x_orig.ndim == 2 or x_orig.ndim == 4):
+            raise ValueError("InputEqualizationObserver only supports Linear and Conv layers")
```

Comment:
We probably need to be able to support conv with 1/2/3 dim as well. Currently looks like this only supports conv2d.